resume-builder-ai/src/app/dashboard/optimizations/[id]/page.tsx

```
@@ -27,6 +28,10 @@ export default function OptimizationPage() {
   const [currentDesignAssignment, setCurrentDesignAssignment] = useState<any>(null);
   const [designLoading, setDesignLoading] = useState(false);
 
+  // Job description data for Apply functionality
+  const [jobDescription, setJobDescription] = useState<any>(null);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)